builder/PythonStubsBuilder.cs

```diff
@@ -0,0 +1,283 @@
+﻿using System;
+using System.Reflection;
+using System.Collections.Generic;
+using System.IO;
+
+namespace PythonStubs {
+    public class PythonStubsBuilderConfigs {
+        public string Prefix { get; set; } = string.Empty;
+        public string Postfix { get; set; } = string.Empty;
+        public bool DestPathIsRoot { get; set; } = false;
+    }
+
+    public static class PythonStubsBuilder {
+        private static List<string> SearchPaths { get; set; } = new List<string>();
+
+        public static string BuildAssemblyStubs(string targetAssemblyPath, string destPath = null, string[] searchPaths = null, PythonStubsBuilderConfigs cfgs = null) {
+            // prepare configs
+            if (cfgs is null)
+                cfgs = new PythonStubsBuilderConfigs();
+
+            // prepare resolver
+            AppDomain.CurrentDomain.AssemblyResolve += AssemblyResolve; ;
```

Comment:
We're adding the same AssemblyResolver over and over every time this function is called.

Reply:
Should I add a static bool flag to mark "AlreadyHandling" or there is a better way?

```csharp
if (!alreadyHandling)
    AppDomain.CurrentDomain.AssemblyResolve += AssemblyResolve;
```

Reply:
You could also just always do a `-=` before a `+=` call